packages/components/src/panel/style.scss

```diff
@@ -42,6 +42,7 @@
 	height: 50px;
 	border-top: $border-width solid $light-gray-500;
 	border-bottom: $border-width solid $light-gray-500;
+	z-index: z-index(".components-panel__header");
```

Comment:
It does feel overkill to have a z-index in the reusable component, can we target the panels inside the inspector instead? or target the inspector container maybe? Not sure what this fixes exactly.

Reply:
We need this so that the header remains on top of the content when it scrolls up.

Reply:
This is on mobile right? could this be applied to the container (sidebar) instead, as I believe we want the whole sidebar to stay above not just the header?

Reply:
>The sidebar content covered the header when scrolling and on mobile it did not scroll at all.

The sidebar content was showing on top of the header. The header is sticky so it needs this to stay on top.

Reply:
makes sense. the style still need to be moved to the stylesheet making the header sticky and not in the generic component.

Reply:
:thinking: It seems to be working without it now. I guess a later change I made fixed it indirectly.